brainstormforce/ultimate-addons-for-gutenberg
# Param: Reset Button - Reset enabled always.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/6bfbc06a-6936-481a-93bb-07910a4400ae?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Improvement

### How has this been tested?

- Please test the below setting - The reset button should be always enabled no matter value is set or not.
  - Typography - Where we used range control for font size, line height.
  - Spacing control - For Padding, Margin.
  - Color control, Border control.
  - Preset - Reset button. It should be enabled always. 
  - Drag and drop 2 or 3 blocks and check the above controls.
- See here - https://share.getcloudapp.com/04uLpZ6e

### Checklist:
